src/openeo_aggregator/backend.py

```diff
@@ -675,74 +675,120 @@ def _get_connection_and_backend_service_id(
         con = self._backends.get_connection(backend_id)
         return con, backend_service_id
 
+    def get_supported_backend_ids(self) -> List[str]:
+        """Get a list containing IDs of the backends that support secondary services."""
+
+        # Try the cache first, but its is cached inside the result of get_service_types().
+        # Getting the service types will also execute querying the capabilities for
+        # which backends have secondary services.
+        return self._get_service_types_cached()["supported_backend_ids"]
+
+    def _find_ids_supported_backends(self) -> List[str]:
+        """Query the backends capabilities to find the ones that support secondary services."""
```

Comment:
It's a bit confusing to have both `get_supported_backend_ids` and `_find_ids_supported_backends`.
I think the latter can be eliminated and just be inlined in the single place where it is used below

Reply:
Done.